src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -136,82 +136,83 @@ public PersistentDataStore() {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         UUID conversationUuid = UUID.fromString((String) entity.getProperty("conv_uuid"));
         UUID authorUuid = UUID.fromString((String) entity.getProperty("author_uuid"));
-        Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        String content = (String) entity.getProperty("content");
-        Message message = new Message(uuid, conversationUuid, authorUuid, content, creationTime);
-        messages.add(message);
+          Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
+          String content = (String) entity.getProperty("content");
+          Message message = new Message(uuid, conversationUuid, authorUuid, content, creationTime);
+          messages.add(message);
       } catch (Exception e) {
-        // In a production environment, errors should be very rare. Errors which may
-        // occur include network errors, Datastore service errors, authorization errors,
-        // database entity definition mismatches, or service mismatches.
-        throw new PersistentDataStoreException(e);
+          // In a production environment, errors should be very rare. Errors which may
+          // occur include network errors, Datastore service errors, authorization errors,
+          // database entity definition mismatches, or service mismatches.
+          throw new PersistentDataStoreException(e);
       }
     }
 
-    return messages;
-  }
-
-  /** Write a User object to the Datastore service. */
-  public void writeThrough(User user) {
```

Comment:
Nothing important is happening here except I moved the functions around to make them look consistent